package.json

```diff
@@ -31,17 +31,24 @@
     "@vue/cli-plugin-e2e-nightwatch": "3.0.0-beta.15",
     "@vue/cli-plugin-eslint": "3.0.0-beta.15",
     "@vue/cli-plugin-unit-jest": "3.0.0-beta.15",
```

Comment:
Shouldn't we remove Jest if we are using Mocha? Or maybe we can use Jest for everything? Could you explain this, it seems strange for me to have 2 test frameworks.

Reply:
Also, please add `coverage` and `.nyc_output` to gitignore

Reply:
It is totally my mistake since Jest is no longer used. I will remove Jest dependencies and config files :)